### ISSUE TITLE
Kodi 19 compatibility

### ISSUE THREAD

**Issue by iz8mbw** [fugkco/script.elementum.jackett#10]
Hello.
Will be possible to add the compatibility for Kodi 19 (Python 3)?
Personally I use Kodi 19 so I cannot use this addon. 
Many thanks!

**Comment by iz8mbw** [comment]
ping @fugkco 
Thanks.

**Comment by fugkco** [comment]
Hi, please feel free to raise a PR. With a baby just born, I've not had time to further develop the add-on.

**Comment by iz8mbw** [comment]
Congratulations, I know well your filing... I have 2 small monsters :-)
Anyway personally no dev skills, so cannot help. 
Can help for test.
If and when you have tine it's appreciated if you can make this addon compatible with Kodi 19. 
Take care.

**Comment by daviddia5** [comment]
I join the request. Thanks!

**Comment by daanvandevliet** [comment]
Hi all, I've just succesfully installed script.elementum.jackett on Kodi 19.

git clone https://github.com/SmellyCat69/script.elementum.jackett.git

I had to update a couple of parts (most importantly the python version):
```[tool.poetry]
name = "script.elementum.jackett"
version = "0.0.1"
description = ""
authors = ["Fugkco <fugkco@github.io>"]
license = "WTFPL"

[tool.poetry.dependencies]
python = "^3.9.5"
requests = "^2.25.1"
requests-toolbelt = "^0.9.1"
torrentool = "^1.0.2"

[tool.poetry.dev-dependencies]
kodistubs = "^19.0.0"
flake8 = "^3.8.2"
pytest = "^4.6.10"
kodi-six = {git = "https://github.com/romanvm/kodi.six.git", rev = "0.1.3"}

[build-system]
requires = ["poetry>=0.12"]
build-backend = "poetry.masonry.api"
````

Then I could run:
poetry update
make all

and I had a .zip with the plugin! (first time I was missing the /resources/lib folder, but on second attempt it worked. Not sure what I changed)

.zip file here: https://github.com/daanvandevliet/script.elementum.jackett/releases/tag/v1.0.0-beta

please consider accepting my PR to get this back up stream.

**Comment by iz8mbw** [comment]
Hi!
Many many thanks for that!!
I have tested this https://github.com/daanvandevliet/script.elementum.jackett/releases/tag/v1.0.0-beta on Kodi 19.1 on Nvidia Shield TV (Android 9, ARM, 64bit) and works well!!!

Thank you!!!!

**Comment by fugkco** [comment]
@daanvandevliet 

> please consider accepting my PR to get this back up stream.

I don't see a PR. Please double check that you haven't raised it in the wrong place.

**Comment by makamekm** [comment]
I have mądre a smalił fix
https://github.com/makamekm/script.elementum.jackett

**Comment by iz8mbw** [comment]
Hi @makamekm what this small fix do?

**Comment by daanvandevliet** [comment]
For those using this, I just made a PR for Kodi v20 to @fugkco . Thank you for your consideration!

**Comment by iz8mbw** [comment]
Thanks.
But when @fugkco will merge?

**Comment by daanvandevliet** [comment]
I'm sure @fugkco will merge once they read this. For now, you can grab the release here: 
https://github.com/daanvandevliet/script.elementum.jackett/releases/tag/v1.0.0-beta-1

Note: I tested install, but not actual function. Please let me know if it works for you!

**Comment by fugkco** [comment]
I've merged @daanvandevliet's PR. Will make a release soon!

**Comment by fugkco** [comment]
I've just created https://github.com/fugkco/script.elementum.jackett/releases/tag/v2.0.0. Please let me know if there are any issues.

**Comment by NicolasThierion** [comment]
Works for me on Nexus. However, it crashes with a TypeError.

I created a PR to fix the issue.

Fixes #4 
Could you please create a new release ?
